src/System Application/App/Edit in Excel/src/Filters/EditInExcelFilters.Codeunit.al

```diff
@@ -48,17 +81,47 @@ codeunit 1490 "Edit in Excel Filters"
     /// <param name="EditInExcelFilterType">The filter type, such as Equal, Greater than etc.</param>
     /// <param name="FilterValue">The value which the field should be Equal to, Greater than etc.</param>
     /// <param name="EditInExcelEdmType">The Edm type of the OData field as specified in the $metadata document.</param>
+    [Obsolete('Use AddFieldV2 instead.', '25.0')]
     procedure AddField(ODataFieldName: Text; EditInExcelFilterType: Enum "Edit in Excel Filter Type"; FilterValue: Text; EditInExcelEdmType: Enum "Edit in Excel Edm Type")
     begin
         EditinExcelFiltersImpl.AddField(ODataFieldName, EditInExcelFilterType, FilterValue, EditInExcelEdmType);
     end;
+#endif
 
+    /// <summary>
+    /// Add the specified field with an initial value using 'and' collection type.
+    /// This is mainly intended for fields that only filter on a single value.
+    /// </summary>
+    /// <param name="ODataFieldName">The OData name of the field referenced.</param>
+    /// <param name="EditInExcelFilterType">The filter type, such as Equal, Greater than etc.</param>
+    /// <param name="FilterValue">The value which the field should be Equal to, Greater than etc.</param>
+    /// <param name="EditInExcelEdmType">The Edm type of the OData field as specified in the $metadata document.</param>
+    procedure AddFieldV2(ODataFieldName: Text; EditInExcelFilterType: Enum "Edit in Excel Filter Type"; FilterValue: Text; EditInExcelEdmType: Enum "Edit in Excel Edm Type")
+    begin
+        EditinExcelFiltersImpl.AddField(ODataFieldName, EditInExcelFilterType, FilterValue, EditInExcelEdmType);
+    end;
+
+#if not CLEAN25
     /// <summary>
     /// Get the field filters for the specified field.
     /// </summary>
     /// <param name="ODataFieldName">The OData name of the field referenced.</param>
     /// <returns></returns>
+    [Obsolete('Use GetV2 instead.', '25.0')]
+#pragma warning disable AL0432
     procedure Get(ODataFieldName: Text): Interface "Edit in Excel Field Filter"
+#pragma warning restore AL0432
+    begin
+        exit(EditinExcelFiltersImpl.Get(ODataFieldName));
+    end;
+#endif
+
+    /// <summary>
+    /// Get the field filters for the specified field.
+    /// </summary>
+    /// <param name="ODataFieldName">The OData name of the field referenced.</param>
+    /// <returns></returns>
+    procedure GetV2(ODataFieldName: Text): Interface "Edit in Excel Field Filter V2"
```

Comment:
We have overloads in AL, so you can just keep this Get(ODataFieldName: Text): Interface "Edit in Excel Field Filter V2"